project/Build.scala

```diff
@@ -108,6 +108,19 @@ object JobServerBuild extends Build {
     dockerfile in docker := {
       val artifact = (assemblyOutputPath in assembly in jobServerExtras).value
       val artifactTargetPath = s"/app/${artifact.name}"
+
+      val sparkBuild = s"spark-$sparkVersion"
+      val sparkBuildCmd = scalaBinaryVersion.value match {
+        case "2.10" =>
+          "./make-distribution.sh -Phadoop-2.4 -Phive"
```

Comment:
Yes, this is a fair point.  We might as well make this Hadoop version configurable.

> On Jun 28, 2016, at 8:21 PM, Noorul Islam K M <notifications@github.com> wrote:
> 
> In project/Build.scala <https://github.com/spark-jobserver/spark-jobserver/pull/530#discussion_r68879916>:
> 
> > @@ -108,6 +108,19 @@ object JobServerBuild extends Build {
> >      dockerfile in docker := {
> >        val artifact = (assemblyOutputPath in assembly in jobServerExtras).value
> >        val artifactTargetPath = s"/app/${artifact.name}"
> > +
> > +      val sparkBuild = s"spark-$sparkVersion"
> > +      val sparkBuildCmd = scalaBinaryVersion.value match {
> > +        case "2.10" =>
> > +          "./make-distribution.sh -Phadoop-2.4 -Phive"
> What if someone wants to compile it with hadoop 2.6?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/530/files/0ccb8703f12d5e70a37a0cc6732c464a6c2ea7bc#r68879916>, or mute the thread <https://github.com/notifications/unsubscribe/ABA32_ATqVfiLvLoI1Dyu_GeSz3ZApcOks5qQeSpgaJpZM4JAAR7>.
>